### ISSUE TITLE
When selecting a chanel to update to, NVDA says already updated no matter which version you

### ISSUE THREAD

**Issue by jcoffin1** [nvda-es/updateChannel#29]
Unable to change update channotel because whenever you do, you can't update to the latest version in that particular channotel. NVDA says "Already updated."

**Comment by patricus3** [comment]
same. sadly
someone has to fix this add-on
